src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -1529,8 +1529,8 @@ void invalidRequestTest(VertxTestContext context) {
                         assertTrue(ar.succeeded());
                         HttpResponse<JsonObject> response = ar.result();
                         HttpBridgeError error = HttpBridgeError.fromJson(response.body());
-                        assertEquals(HttpResponseStatus.BAD_REQUEST.code(), response.statusCode());
-                        assertEquals(HttpResponseStatus.BAD_REQUEST.code(), error.getCode());
+                        assertEquals(HttpResponseStatus.NOT_FOUND.code(), response.statusCode());
```

Comment:
Really you ought to rename this `invalidRequestTest()` method to something like `postToNonexistentEndpoint()`, because that's what it's doing. In reality there are many kinds of invalid request, but this method is just testing a single case.